File: android/src/main/java/com/github/musicode/codeview/RNTCodeViewModule.java

package com.github.musicode.codeview;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;

public class RNTCodeViewModule extends ReactContextBaseJavaModule {

  private final ReactApplicationContext reactContext;

  public RNTCodeViewModule(ReactApplicationContext reactContext) {
    super(reactContext);
    this.reactContext = reactContext;
  }

  @Override
  public String getName() {
    return "RNTChatView";
  }

}

File: android/src/main/java/com/github/musicode/codeview/RNTCodeViewModule.kt
package com.github.musicode.codeview

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule

class RNTCodeViewModule(private val reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {

    override fun getName(): String {
        return "RNTCodeView"
    }

}